### PR TITLE
games-arcade/savagewheels: New ebuild.

### PR DESCRIPTION
**Pull request by nvinson**
This will add savagewheels-1.6.0.

Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by nvinson** [comment]
Given the large patchset, an issue has been filed upstream to get equivalent changes added (https://github.com/petarov/savagewheels/issues/6).

**Comment by nvinson** [comment]
@gentoo/proxy-maint

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

**Comment by idella** [comment]
@kensington

File: games-arcade/savagewheels/savagewheels-1.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+GAMEDATA="${PN}-gamedata-1.4.0"
+
+SRC_URI="
+	https://github.com/petarov/savagewheels/archive/v1.6.0.tar.gz -> ${P}.tar.gz
+	https://github.com/petarov/savagewheels/releases/download/v1.4/${PN}-gamedata.tar.gz -> ${GAMEDATA}.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64" IUSE="debug fmod sound"
```
Comment (gktrk) [inline games-arcade/savagewheels/savagewheels-1.6.0.ebuild]: Doesn't IUSE usually start on a new line?

Comment (nvinson) [inline games-arcade/savagewheels/savagewheels-1.6.0.ebuild]: IUSE should be on its own line.  I'll fix it within 7 days.  Thanks.

File: games-arcade/savagewheels/savagewheels-1.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+DEPEND="
```
Comment (nvinson) [inline games-arcade/savagewheels/savagewheels-1.6.0.ebuild]: s/DEPEND/RDEPEND

File: games-arcade/savagewheels/savagewheels-1.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+	media-libs/libsdl
+	sound? (
+		!fmod? ( media-libs/sdl-mixer[mod,modplug] )
+		fmod? ( >=media-libs/fmod-4.38.00 )
+	)"
+
+RDEPEND="${DEPEND}"
```
Comment (nvinson) [inline games-arcade/savagewheels/savagewheels-1.6.0.ebuild]: change line to DEPEND="${RDEPEND}"

**Comment by monsieurp** [comment]
```
>>> Unpacking source...
>>> Unpacking savagewheels-1.6.0.tar.gz to /var/tmp/portage/games-arcade/savagewheels-1.6.0/work
>>> Unpacking savagewheels-gamedata-1.4.0.tar.gz to /var/tmp/portage/games-arcade/savagewheels-1.6.0/work/savagewheels-gamedata-1.4.0
>>> Source unpacked in /var/tmp/portage/games-arcade/savagewheels-1.6.0/work
>>> Preparing source in /var/tmp/portage/games-arcade/savagewheels-1.6.0/work/savagewheels-1.6.0 ...
 * Applying savagewheels-1.6.0-cmakelists.txt.patch ...                                                                                                                [ ok ]
 * Applying savagewheels-1.6.0-linux_dirs.patch ...
3 out of 3 hunks FAILED -- saving rejects to file src/Main.cpp.rej
3 out of 3 hunks FAILED -- saving rejects to file src/Main.h.rej
2 out of 2 hunks FAILED -- saving rejects to file src/CGame.cpp.rej
1 out of 1 hunk FAILED -- saving rejects to file src/CSdl.cpp.rej
1 out of 1 hunk FAILED -- saving rejects to file src/CSounds.cpp.rej
1 out of 1 hunk FAILED -- saving rejects to file src/CSwv_module.cpp.rej
1 out of 1 hunk FAILED -- saving rejects to file src/CSdl.cpp.rej
1 out of 1 hunk FAILED -- saving rejects to file src/CMainMenu.cpp.rej
1 out of 1 hunk FAILED -- saving rejects to file src/CMainMenu.cpp.rej                                                                                                 [ !! ]
 * ERROR: games-arcade/savagewheels-1.6.0::gentoo failed (prepare phase):
 *   patch -p1  failed with /home/monsieurp/gentoo/games-arcade/savagewheels/files/savagewheels-1.6.0-linux_dirs.patch
```

Please make sure patches correctly apply.

**Comment by nvinson** [comment]
That's odd.  Works fine for me.

```
% ebuild savagewheels-1.6.0.ebuild prepare
Appending /data/nvinson/gentoo to PORTDIR_OVERLAY...
 * savagewheels-1.6.0.tar.gz SHA256 SHA512 WHIRLPOOL size ;-) ...                                                [ ok ]
 * savagewheels-gamedata-1.4.0.tar.gz SHA256 SHA512 WHIRLPOOL size ;-) ...                                       [ ok ]
>>> Unpacking source...
>>> Unpacking savagewheels-1.6.0.tar.gz to /var/tmp/portage/games-arcade/savagewheels-1.6.0/work
>>> Unpacking savagewheels-gamedata-1.4.0.tar.gz to /var/tmp/portage/games-arcade/savagewheels-1.6.0/work/savagewheels-gamedata-1.4.0
>>> Source unpacked in /var/tmp/portage/games-arcade/savagewheels-1.6.0/work
>>> Preparing source in /var/tmp/portage/games-arcade/savagewheels-1.6.0/work/savagewheels-1.6.0 ...
 * Applying savagewheels-1.6.0-cmakelists.txt.patch ...                                                          [ ok ]
 * Applying savagewheels-1.6.0-linux_dirs.patch ...                                                              [ ok ]
>>> Source prepared.
```

**Comment by monsieurp** [comment]
Alright. I think my setup has a bunch of missing libs. Anyway, I'll merge it.

**Pull request closed without merging**